<launch>

    <arg name="param_path" default="$(find rtk_localization)/cfg/huaxian_0602.txt"/>
    <arg name="param_path_data" default="$(find rtk_localization)/data/huaxian_0602.txt"/>
    <arg name="gps_topic" default="/chcnav_ros/gnss_ins"/>

    <node pkg="rtk_localization" type="map_pub" name="map_pub_node" output="screen">
        <param name="param_path" value="$(arg param_path)"/>
        <param name="param_path_data" value="$(arg param_path_data)"/>
        <param name="gps_topic" value="$(arg gps_topic)"/>
    </node>
    <!-- <node pkg="control" type="control" name="control" output="screen"/> -->
    <include file="$(find rtk_localization)/launch/rtk_loc_hxc.launch"/>
    
</launch>
